src/app/Dashboard/Diagnostics/DiagnosticsCard.tsx

```diff
@@ -58,6 +64,30 @@ export const DiagnosticsCard: DashboardCardFC<DiagnosticsCardProps> = (props) =>
     [notifications, t],
   );
 
+  React.useEffect(() => {
+    addSubscription(
+      serviceContext.target
+        .target()
+        .pipe(
+          filter((target) => !!target),
+          first(),
+          concatMap(() => serviceContext.api.getThreadDumps()),
+        )
+        .subscribe({
+          next: (dumps) => (dumps.length > 0 ? setThreadDumpReady(true) : setThreadDumpReady(false)),
+          error: () => setThreadDumpReady(false),
+        }),
+    );
+  });
```

Comment:
This hook is missing a dependencies array - it's likely the cause of, or related to, the multiple queries bug I described above.